engine/src/main/java/com/sandboni/core/engine/render/banner/BannerRenderService.java

```diff
@@ -33,7 +33,7 @@ private String bannerToDisplay() {
                 builder.append(l).append(N_LINE);
             }
         } catch (IOException e) {
-            log.warn("Error in rendering Sandboni banner : {} , {}", LOGO_FILE_NAME, e);
+            log.warn(String.format("Error in rendering Sandboni banner : %s , %s", LOGO_FILE_NAME, e.getMessage()), e);
```

Comment:
why changing?

Reply:
sonar failed on old code

Reply:
Are you sure that you are fixing the correct sonar warning? I don't think that Sonar complains about the {} format, this is pretty standard and we use it across all our loggers. Seems to me that is complaining about the exception. Please double check.

Reply:
I've rolled this back so you can have a look